src/components/component.ts

```diff
@@ -233,10 +234,19 @@ export class Component {
    * Renders the Component without waiting for the next frame.
    */
   public renderImmediately() {
+    Utils.Window.setTimeout(() => this._onRenderCallbacks.callCallbacks(this), this._deferredTotalDrawTime());
```

Comment:
yeah, I don't mean adding an unit test, but I would just want to know if it's verified that this works as setTimeout is tricky